provider/k8s.go

```diff
@@ -496,11 +497,28 @@ func (store *genericFileStore) DeleteAll(dir string) error {
 }
 
 func (store *genericFileStore) Write(key string, data []byte) error {
-	err := store.bucket.WriteAll(context.TODO(), key, data, nil)
+	ctx := context.TODO()
+	err := store.bucket.WriteAll(ctx, key, data, nil)
 	if err != nil {
 		return err
 	}
-	return nil
+	err = re.Do(
+		func() error {
+			blob, errRetr := store.bucket.ReadAll(ctx, key)
```

Comment:
This may be a lot if the file is large, but well see when we get there

Reply:
Agreed. I can check the docs again to see if there's another method that might give us the same assurance without reading in the entire contents. One more thought: I wonder if this is _as_ important given the client-side changes I put in place to ensure the call to `apply` in `client.dataframe` is synchronous and waits for the resource to either be in a Ready or Failed state. Thouhts?